packages/network/test/integration/proxy-connections.test.ts

```diff
@@ -307,6 +372,5 @@ describe('Proxy connection tests', () => {
                 signature: 'signature'
             }))
         ])
-    }, 20100)
-
+    }, 60000)
```

Comment:
Does the connection process take significantly longer than previously?

Reply:
The test was intentionally made to last longer to simulate a longer downtime from the server. The test could be made faster by adding configurability to the reconnection interval